doc/publishing-api-syntactic-usage.md

```diff
@@ -68,6 +68,35 @@ Requests to update an existing draft content item:
  - `locale` (optional) specifies the locale of the content item to be published.
  - `previous_version` (optional but advised) is used to ensure the request is publishing the latest lock version of this draft. ie. optimistic locking.
 
+ ## `POST /v2/content/:content_id/unpublish`
+
+ [Request/Response detail]()
+
+  - Will refuse to unpublish a lone draft.
+  - Will refuse to unpublish a redrafted document unless `discard_drafts` is `true`.
+  - Validates that unpublishing `type` is one of `withdrawal`, `gone` or `redirect` and raises a 422 otherwise.
+  - Retrieves the live content item with the matching content_id and locale and changes its state to `unpublished`.
+  - Creates an `Unpublishing` with the provided details.
+  - Will update the `Unpublishing` if the document is already unpublished.
+  - Sends the gone/redirect/withdrawal to the live content store.
+  - Does not send to the draft content store (unless a draft was discarded).
+  - Does not send to the message queue.
+  - Returns 200 along with the content_id of the unpublished item.
```

Comment:
Would it be worth re-presenting the whole content item rather than just returning the content_id? We've used this pattern for other things (e.g. PutContent) and the ContentItemPresenter is really fast now.